client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -617,22 +630,26 @@ private ConcurrentHashMap<Integer, PartitionLocation> registerShuffleInternal(
           }
           return result;
         } else if (StatusCode.SLOT_NOT_AVAILABLE.equals(respStatus)) {
+          lastException = new CelebornIOException("Slot not available for shuffle " + shuffleId);
```

Comment:
Can these be just error message string, which we can later pass as params to the final throw statement?

Reply:
@jiang13021, @s0nskar, IMO, the exception message could use the status code with shuffle id.

Reply:
@s0nskar @SteNicholas  Fixed